functions/src/index.ts

```diff
@@ -49,4 +50,23 @@ async function setAdminPriveleges(userEmail: string): Promise<void> {
       isPartner: false,
     });
   }
-}
\ No newline at end of file
+}
+
+export const isAdmin = functions.https.onRequest((request: any, response: any) => {
```

Comment:
do we need a function/button for this? can we just run a script on the page that checks it for the user?